### PR TITLE
Styling/header

### PR DESCRIPTION
**Pull request by bradybridges**

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -1,65 +1,145 @@
 import { Link } from "gatsby"
 import PropTypes from "prop-types"
-import React from "react"
-import logo from "../images/logo.svg";
+import React, { useEffect, useRef, useState } from "react"
+import logo from "../images/logo.svg"
```
Comment (LayaTaal) [inline src/components/header.js]: I don't think we are using this here, so we can remove the import.

File: src/components/header.js [hunk 2]
```diff
@@ -1,65 +1,145 @@
...
+                News
+              </Link>
+            </li>
+            <li className="nav-list__item nav-list-mobile__item">
+              <Link className={linkStyles} to="/contact">
+                Contact
+              </Link>
+            </li>
+          </ul>
+        </nav> */}
```
Comment (LayaTaal) [inline src/components/header.js]: Don't forget to remove this.

File: src/components/hamburgerMenu.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  const inactiveClass = "hamburger hamburger--slider"
+  const activeClass = "hamburger hamburger--slider is-active"
+
+  const handleClick = () => {
+    setActive(!active)
+
+    if (!active) {
+      headerRef.current.classList.add("header--secondary")
+      headerRef.current.classList.remove("header--primary")
+      mobileNavRef.current.style.height = "500px"
```
Comment (LayaTaal) [inline src/components/hamburgerMenu.js:34]: I think we can achieve this by targeting a class name and using breakpoints instead. That will probably help with the issue when resizing back and forth between mobile and desktop.

**Review by LayaTaal — APPROVED** [review]
Ok looks really good @bradybridges! Once we go over the updates to accessibility and have those merged in along with a few touch ups here, let's get this bad boy merged!

File: src/components/header.js (hunk 2 above)
Comment (bradybridges) [reply src/components/header.js]: Opened an issue for this! Since targeting classNames breaks the animations.

**Pull request merged**